packages/design-system-react/src/components/button-primary/index.ts

```diff
@@ -0,0 +1,3 @@
+export { ButtonPrimary } from './ButtonPrimary';
+export type { ButtonPrimaryProps } from './ButtonPrimary.types';
+export { ButtonBaseSize as ButtonPrimarySize } from '../button-base';
```

Comment:
Here we are using an alias for ButtonBaseSize this ensures a source of truth and alignment with the component